priv/perf/apps/load_test/lib/application.ex

```diff
@@ -15,10 +19,38 @@ defmodule LoadTest.Application do
         max_connections: max_connections
       )
 
+    :ok = start_services()
+
     {:ok, self()}
   end
 
   def stop(_app) do
     :hackney_pool.stop_pool(:perf_pool)
   end
+
+  defp start_services() do
+    {:ok, _} =
+      Supervisor.start_link([{LoadTest.Service.NonceTracker, []}], name: LoadTest.Supervisor, strategy: :one_for_one)
```

Comment:
@Ino I keep getting 
```
warning: :simple_one_for_one strategy is deprecated, please use DynamicSupervisor instead
```
when running the app via `MIX_ENV=test iex -S mix run`
The thing is, when I change it to `DynamicSupervisor` I get the same warning. Maybe I do something wrong.
Could you take a look on that?

Reply:
I'm not exactly sure what you're trying to do that you would need `one_for_one `. 
For what is worth, `NonceTracker` could really be a simple implementation over a public ETS or https://erlang.org/doc/man/counters.html